sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSender.cs

```diff
@@ -66,7 +66,8 @@ public virtual bool IsClosed
         /// Gets the ID to identify this client. This can be used to correlate logs and exceptions.
         /// </summary>
         /// <remarks>Every new client has a unique ID.</remarks>
-        internal string Identifier { get; private set; }
+        ///
```

Comment:
nit:  We should consider updating this block to remove the implied guarantee about uniqueness, since we don't have control over the value in all cases.